src/threadblock/element_binary.cc

```diff
@@ -77,6 +77,13 @@ TBOperator *Graph::create_elementbinary_op(STensor const &input1,
   } else {
     return op;
   }
+<<<<<<< HEAD
```

Comment:
This is a git merge conflict you will need to fix. Make sure you can compile the code after the merge.